sample-app/src/components/DecoratedAppliedFilters.tsx

```diff
@@ -3,23 +3,25 @@ import { AppliedQueryFilter } from "@yext/answers-core";
 import { useAnswersState } from '@yext/answers-headless-react';
 import { GroupedFilters } from '../models/groupedFilters';
 import { getGroupedAppliedFilters } from '../utils/appliedfilterutils';
+import { StaticFiltersLabelConfig } from "./StaticFilters";
 
 export interface DecoratedAppliedFiltersConfig {
   showFieldNames?: boolean,
   hiddenFields?: Array<string>,
   labelText?: string,
   delimiter?: string,
+  staticFiltersGroupLabels?: Record<string, StaticFiltersLabelConfig>,
```

Comment:
Would it be simpler if this was just a mapping of fieldId to label? instead of a mapping of fieldId to an object with a single label property

Reply:
Yes, that would be simpler. I did it this way to make it more clear what the mapping was between, but I could also just do that with documentation instead

Reply:
Ah ok, let's just go with the string to string mapping for now, that way people don't have to understand what a StaticFiltersLabelConfig is, if needed we can tweak it later

Reply:
Updated!